scripts/release_scripts/update_changelog_and_credits.py

```diff
@@ -506,6 +506,27 @@ def update_version_in_config_files():
     )
 
 
+def inform_server_errors_team(release_rota_url, server_error_playbook_url):
+    """Asks the release co-ordinator to inform the server errors team
+    that the release will be deployed to production.
+
+    Args:
+        release_rota_url: str. URL pointing to the page which lists the rota
+            to be followed for release coordinators.
+        server_error_playbook_url: str. URL pointing to the server error team
+            playbook.
+    """
+    common.open_new_tab_in_browser_if_possible(release_rota_url)
+    common.open_new_tab_in_browser_if_possible(server_error_playbook_url)
+    common.ask_user_to_confirm(
+        'Ping the Server errors team coordinator (you can find them here: %s) '
+        'that the release will be deployed on the production server and '
+        'send them the Playbook for the Server Errors Team: %s.'
+        'Wait for them to confirm they have done all the stuff that was needed '
+        'before doing the deployment.' % (
```

Comment:
I think we can add something like "if there is no response in 24 hours then go ahead with the deployment"

Reply:
Done.